py_v_sdk/contract/atomic_swap_ctrt.py

```diff
@@ -151,14 +243,331 @@ async def token_id(self) -> str:
         """
         return await self._query_db_key(self.DBKey.for_token_id())
 
-    async def get_tok_bal(self, addr: str) -> int:
+    async def get_swap_balance(self, addr: str) -> md.Token:
         """
-        get_tok_bal queries & returns the balance of the token deposited into the contract.
+        get_swap_balance queries & returns the balance of the token deposited into the contract.
 
         Args:
             addr (str): The account address that deposits the token.
 
         Returns:
-            int: The balance of the token.
+            md.Token: The balance of the token.
+        """
+        bal = await self._query_db_key(self.DBKey.for_token_balance(addr))
+
+        resp = await self.chain.api.ctrt.get_tok_info(await self.token_id)
+        unit = resp["unity"]
+        return md.Token.for_amount(bal, unit)
+
+    async def get_swap_owner(self, tx_id: str) -> md.Addr:
+        """
+        get_swap_owner queries & returns the address of swap owner.
+
+        Args:
+            tx_id (str): The lock transaction id.
+
+        Returns:
+            md.Addr: The address of the swap owner.
+        """
+        owner_addr = await self._query_db_key(self.DBKey.for_swap_owner(tx_id))
+
```

Comment:
This blank line before the return statement can be removed. The same for other DB querying methods like `get_XXX`